lld/wasm/Driver.cpp

```diff
@@ -94,6 +94,9 @@ class LinkerDriver {
   // True if we are in --whole-archive and --no-whole-archive.
   bool inWholeArchive = false;
 
+  // True if we are in --start-lib and --end-lib.
```

Comment:
```suggestion
  // True if we are between --start-lib and --end-lib.
```

Reply:
Just matching the comment above and in the ELF linker here.  I'm not sure this is a huge grammatical error.